contrib/kind.sh

```diff
@@ -293,6 +336,16 @@ install_j2_renderer() {
 }
 
 check_dependencies() {
+  if ! command_exists curl ; then
+    echo "Dependency not met: Command not found 'curl'"
+    exit 1
+  fi
+
+  if ! command_exists kubectl ; then
+    echo "Dependency not met: Command not found 'kubectl'"
```

Comment:
Could you log that you are installing it, similar to how it is done with j2 below?

Reply:
@jcaamano done, thanks for the review.